.npmignore

```diff
@@ -2,3 +2,7 @@
 
 tsconfig.json
 tslint.js
+.circleci
+*.ts
+!*.d.ts
+!schematics/page/files/**/*.ts
```

Comment:
This could be `!schematics/*/files/**/*.ts` to avoid any accidental exclusion of `.ts` files for new schematics.

Reply:
Updated